test/Dialect/FIRRTL/grand-central.mlir

```diff
@@ -85,11 +85,12 @@ firrtl.circuit "InterfaceGroundType" attributes {
 // CHECK-SAME:   @InterfaceGroundType
 // CHECK-SAME:   #hw.innerNameRef<@InterfaceGroundType::@dut>
 // CHECK-SAME:   #hw.innerNameRef<@DUT::@b>
-// CHECK-NEXT: sv.verbatim "assign {{[{][{]0[}][}]}}.baz = {{[{][{]1[}][}]}}.{{[{][{]2[}][}]}}.{{[{][{]3[}][}]}}[1].d;"
+// CHECK-NEXT: sv.verbatim "assign {{[{][{]0[}][}]}}.baz = {{[{][{]1[}][}]}}.{{[{][{]2[}][}]}}.{{[{][{]3[}][}]}}{{[{][{]4[}][}]}};"
```

Comment:
In the module, an XMR followed by a index op with the result passed to the verbatim would be equivalent and not require new operations.

Reply:
Or perhaps more appropriately an subindex/subaccess in the referenced module feeding a hw.probe.